docs/CHANGELOG.md

```diff
@@ -18,6 +18,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Arbitrum` chains are no longer restricted to only `FixedPrice` `GAS_ESTIMATOR_MODE`
 - Updated `Arbitrum Rinkeby` configuration for Nitro
 - `chainlink admin users update` command is replaced with `chainlink admin users chrole` (only the role can be changed for a user)
+- It is now possible to use the same key across multiple chains.
+- The `setnextnonce` local client command has been removed, and replaced by a more general key/chain client command.
+
+### NEW `evm keys chain` command
+
+This can also be accessed at `/v2/keys/eth/chain`.
```

Comment:
Not `evm`?
```suggestion
This can also be accessed at `/v2/keys/evm/chain`.
```

Reply:
Confusingly, no. All API endpoints are still `eth`.

Reply:
I guess it matches the local style :shrug:

Reply:
Added new routes and changed.